CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Changes since v2.20
 
 ### Additions
 - "More info" support for DOI style resources (when `:enable-doi` flag is set) (#2701)
+- Entitlement expiration date field in application actions will now accept only future dates (#2674)
```

Comment:
This is sort of wrong isn't it? I mean calendar vs. text field / API usage? We could be very precise here but let's see if we should implement the rest of the possibilities too (see review comment).

Reply:
yes you are right - i'll update the message to mention that it concerns the front-end calendar specifically